src/computer.cpp

```diff
@@ -631,69 +634,6 @@ void computer::activate_function( computer_action action )
         }
         break;
 
```

Comment:
There is no `default` statement in `switch` and `COMPACT_OBSOLETE` case is missing too, so most compilers would complain about that.

```suggestion

        case COMPACT_OBSOLETE:
            break;
```